component/service/src/main/java/org/exoplatform/social/rest/api/SpaceRestResources.java

```diff
@@ -62,6 +63,7 @@ public abstract Response getSpaces(@Context UriInfo uriInfo,
                                      @QueryParam("order") String order,
                                      @QueryParam("returnSize") boolean returnSize,
                                      @QueryParam("favorites") boolean favorites,
+                                     @QueryParam("unread") boolean unread,
```

Comment:
```suggestion
```
Please avoid adding a dedicated parameter to retrieve or the unread information, we can use expand parameter.

Reply:
Done!